snowflake/src/main/java/org/apache/iceberg/snowflake/JdbcSnowflakeClient.java

```diff
@@ -125,6 +129,13 @@ public <T> T query(Connection conn, String sql, ResultSetParser<T> parser, Strin
   private final JdbcClientPool connectionPool;
   private QueryHarness queryHarness;
 
+  protected static final Set<Integer> DATABASE_NOT_FOUND_ERROR_CODES =
+      Sets.newHashSet(2001, 2003, 2043);
```

Comment:
`ImmutableSet.of(2001, 2003, 2043)` is usually more idiomatic for these patterns.

Reply:
As discussed, since it's already repackaged guava, we'd just be using the next-door-neighbor of Sets:

    import org.apache.iceberg.relocated.com.google.common.collect.ImmutableSet;